src/index.test.js

```diff
@@ -1,40 +1,27 @@
 import fs from 'fs';
 import Chromy from 'chromy';
+import getPort from 'get-port';
 import Differencify from './index';
 import logger from './logger';
 
-let chromyCloseCallsCounter = 0;
-jest.mock('chromy', () => jest.fn(() =>
```

Comment:
You can probably get away with only close being mocked here, since we're mocking chrome runner which calls the other methods.